File: 1944-truncate-sentence/truncate-sentence.cs
public class Solution {
    public string TruncateSentence(string s, int k) {
        List<string> tr = s.Split(' ').ToList();
        List<string> trn = new List<string>();
        for(int i=0; i<k; i++) 
        {
            trn.Add(tr[i]);
        }
        return string.Join(' ', trn);
    }
}

File: 1944-truncate-sentence/truncate-sentence.py
class Solution(object):
    def truncateSentence(self, s, k):
        """
        :type s: str
        :type k: int
        :rtype: str
        """
        a = []
        s = list(s.split())
        for i in range(k):
            a += [s[i]]
        return " ".join(a)